requests_cache/response.py

```diff
@@ -160,6 +173,23 @@ def stream(self, amt=None, **kwargs):
 AnyResponse = Union[Response, CachedResponse]
 
 
+def format_datetime(value: Optional[datetime]) -> str:
+    return value.strftime(DATETIME_SHORT_FORMAT) if value else "N/A"
+
+
+def format_file_size(n_bytes: int) -> str:
+    """Convert a file size in bytes into a human-readable format"""
+    filesize = float(abs(n_bytes))
+
+    def _format(unit):
+        return f'{int(filesize)} {unit}' if unit == 'bytes' else f'{filesize:.2f} {unit}'
+
+    for unit in ['bytes', 'KB', 'MB', 'GB']:
```

Comment:
```suggestion
    for unit in ['bytes', 'KiB', 'MiB', 'GiB']:
```

Reply:
Good catch.

Reply:
And do note your other code relies on those values. I think I saw compare to GB somewhere.